images/dcos/bootstrap.sh

```diff
@@ -220,7 +220,7 @@ EOF
 #
 # nginx, proxying the apiserver and serving kubectl binaries
 #
-sed "s,<PORT>,${apiserver_proxy_port},;s,<APISERVER>,http://${host_ip}:${apiserver_port}," /etc/nginx/conf.d/default.conf.template > /etc/nginx/conf.d/default.conf
+sed "s,<PORT>,${apiserver_proxy_port},;s,<APISERVER>,https://${host_ip}:${apiserver_secure_port}," /etc/nginx/conf.d/default.conf.template > /etc/nginx/conf.d/default.conf
```

Comment:
AFAIK Nginx does not support SPDY without TLS. But it would be worth further investigation.

In the other comment I meant that k8s is able to upgrade a HTTP connection to SPDY.